packages/cws-trx/src/index.ts

```diff
@@ -1,89 +1,115 @@
-import {
-	apdu,
-	coin as COIN,
-	transport as tp,
-	error,
-	tx,
-} from '@coolwallet/core';
+import { coin as COIN, } from '@coolwallet/core';
 import * as trxSign from './sign';
-import { SignTransactionData } from './config/type';
-import * as scriptUtil from './utils/scriptUtils';
-import * as trxUtil from './utils/trxUtils';
-import * as scripts from './config/scripts';
-
-type Transport = tp.default;
+import * as scriptUtil from './utils/scriptUtil';
+import * as txUtil from './utils/transactionUtil';
+import * as type from './config/types';
+import { TX_TYPE } from './config/types';
+import * as params from './config/params';
 
+export { TX_TYPE };
 export default class TRX extends COIN.ECDSACoin implements COIN.Coin {
 	constructor() {
-		super('C3');
+		super(params.COIN_TYPE);
 	}
 
 	/**
 	 * Get Tron address by index
 	 */
 	async getAddress(
-		transport: Transport,
+		transport: type.Transport,
 		appPrivateKey: string,
 		appId: string,
 		addressIndex: number
 	): Promise<string> {
 		const publicKey = await this.getPublicKey(transport, appPrivateKey, appId, addressIndex);
-		return trxUtil.pubKeyToAddress(publicKey);
+		return txUtil.pubKeyToAddress(publicKey);
 	}
 
 	/**
 	 * Sign Tron Transaction.
 	 */
-	async signTransaction(signTxData: SignTransactionData): Promise<string> {
+	async signTransaction(signTxData: type.NormalTradeData): Promise<string> {
 		const {
-			transport, transaction, appPrivateKey, appId, addressIndex, confirmCB, authorizedCB
+		  transport, transaction, appPrivateKey, appId, addressIndex, confirmCB, authorizedCB
 		} = signTxData;
 
+		const script = params.TRANSFER.script + params.TRANSFER.signature;
+		const argement = await scriptUtil.getNormalTradeArgument(transaction, addressIndex);
```

Comment:
should be `argument` or simply `arg` if we want to type less. Same issue in other getArgument functions.